predict.py

```diff
@@ -111,12 +127,18 @@ def main(
     dry_run: bool,
     from_model: Optional[str],
     verbose: bool,
-    max_assets: Optional[int]
+    max_assets: Optional[int],
 ):
 
     kili = Kili(api_key=api_key)
     input_type, jobs = get_project(kili, project_id)
-    assets = get_assets(kili, project_id, label_types.split(","), max_assets = max_assets)
+    assets = get_assets(
+        kili,
+        project_id,
+        label_types.split(","),
+        max_assets=max_assets,
+        only_labeled=True,
```

Comment:
Good idea to let the possibility to select the unlabeled assets. However I think we should define how `label_types` and `only_labeled` are articulated. I propose the following args:
```
* get_labeled=True,
* get_unlabeled=True,
* label_types_in=["DEFAULT", "REVIEW"] (ignored if get_labeled = False)
```
if `get_unlabeled==False` and `get_labeled=False`, this should crash.
WDYT?

Reply:
I agree, doing it